conversion.py

```diff
@@ -7,4 +7,7 @@ def celsius_to_kelvin(celsius):
 def kelvin_to_celsius(kelvin):
     return kelvin - 273.15
 
+def kelvin_to_fahrenheit(kelvin):

```

Comment:
good , you can reuse your previous function
```suggestion
def kelvin_to_fahrenheit(kelvin):
```python
return celsius_to_fahrenheit( kelvin_to_celsius(kelvin) )

``` 
```